src/Landing/index.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import FacebookButton from "../auth/socialAuth/facebook/components/Facebook";
+import GoogleButton from "../auth/socialAuth/google/components/Google";
+import axios from "axios";
+
+let errorMessage = "";
+
+export default class SocialAuthentication extends Component {
+  responseFacebook = response => {
+    const config = {
+      headers: {
+        "Content-Type": "application/json"
+      },
+      responseType: "json"
+    };
+
+    axios
+      .post(
+        "https://ah-backend-realers-staging.herokuapp.com/api/auth/facebook/",
+        {
+          user: {
+            access_token: response.accessToken
+          }
+        },
+        config
+      )
+      .then(response => {
+        localStorage.setItem("access_token", response.data.user.token);
+        document.location.href = "/";
+      })
+      .catch(() => {
+        errorMessage = "Error Authenticating with Facebook, check your connection";
+      });
+  };
+
+  responseGoogle = response => {
+    const config = {
+      headers: {
+        "Content-Type": "application/json"
+      },
+      responseType: "json"
+    };
+
+    axios
+      .post(
+        "https://ah-backend-realers-staging.herokuapp.com/api/auth/google/",
+        {
+          user: {
+            access_token: response.Zi.id_token
+          }
+        },
+        config
+      )
+      .then(response => {
+        localStorage.setItem("access_token", response.data.user.token);
+        document.location.href = "/";
```

Comment:
Use the router... I have a feeling if you do this all the state is lost. So, you can do something like so `this.props.history.push('/')`
Since routing is set up for the app, by default react-router passes a history prop to your components and you can access that prop like any other. 
Usually, if a component is nested, the prop may not be defined. That way, you can wrap your component with `withRouter`

Reply:
@AGMETEOR I have worked on this but in a different way please check it out.